tests/test_helpers.py

```diff
@@ -21,10 +21,8 @@
       "-a","Kurt Godel", "MC Escher", "Johann Sebastian Bach","-c", "Bitcoin",
       "--other_agencies", "Flatland","-n", "this is a sample added proposal"],
      "20_anewproposal has been added in proposals\n"),
-    (["helper", "a_expense", "159.18", "Payment to google", "-b", "-g", "dmref15", "mrsec14"],
-     "Payment to google has been added to expenses\n"),
-    (["helper", "a_expense", "159.18", "Payment to google", "-b", "-y",
-      "ashaaban", "-g", "dmref15", "mrsec14"],
+    (["helper", "a_expense", "159.18", "google", "Payment to Google Cloud to pay for cloud services",
+      "-b", "g", "dmref15", "mrsec14", "-d", "2020-06-18"],
```

Comment:
I think this should be -g.  Also, to get more test coverage, add something for all the optional variables and write the test for the travel case, which is more complicated.

Reply:
done